config/bootstrap.php

```diff
@@ -89,7 +89,7 @@
  */
 Configure::write('debug', 2);
 if (Configure::read('debug')) {
-    // Configure::write('DebugKit.forceEnable', TRUE);
```

Comment:
add this to gitignore..
create bootstrap.local.php  -> this will serve as your local settings
to be rename bootstrap.php by other devs